### PR TITLE
SCAT-2783 new order field to question

### PR DESCRIPTION
**Pull request by sudhakar-reddy-konda**

### CODE REVIEW

File: cat/CaT-service.yaml [hunk 1]
```diff
@@ -3877,6 +3877,10 @@ components:
               description: |+
                 Defines a the buyer can add new questions of this type
               type: boolean
+            order:
```
Comment (tommyb82) [inline cat/CaT-service.yaml:3880]: Should we also add this to `QuestionGroup.nonOCDS`?  I think it was the group display that is highlighted in https://crowncommercialservice.atlassian.net/browse/SCAT-2783 🤔

**Review by tommyb82 — CHANGES_REQUESTED** [review]
Looks good but might need it on `QuestionGroup`?

File: cat/CaT-service.yaml (hunk 1 above)
Comment (sudhakar-reddy-konda) [reply cat/CaT-service.yaml:3880]: yes Agree, guess same for questions with in group. it need order field at group and question as well

**Review by tommyb82 — APPROVED** [review]
👍

**Pull request merged**